packages/web-components/readme.md

```diff
@@ -16,22 +21,19 @@ Stencil components are just Web Components, so they work in any major framework
 
 ## Getting Started
 
-To start building a new web component using Stencil, clone this repo to a new directory:
-
-```bash
-git clone https://github.com/ionic-team/stencil-component-starter.git my-component
-cd my-component
-git remote rm origin
-```
+To start building a new web component using Stencil:
 
-and run:
+run:
 
 ```bash
 npm install
-npm start
+npm serve
 ```
+running `npm serve` will start the dev server so the new component can be viewed in the browser. 
+
+After creating and naming your component, add it to `./index.html` so the dev server picks it up. Remove the `<my-component />` dummy component from index.html 
```

Comment:
This paragraph is unnecessary if using storybook for development and testing.

Reply:
Agree, but I think both ways are useful. I'd like to keep the guidance for working only with the stencil dev server. See comment above for adding some words that link over to the steps in the monorepo readme.